test/extended/conformance.sh

```diff
@@ -25,11 +25,6 @@ ps=$(join '|' "${parallel_exclude[@]}")
 sf=$(join '|' "${serial_only[@]}")
 ss=$(join '|' "${serial_exclude[@]}")
 
-echo "[INFO] Running the following tests:"
-TEST_REPORT_DIR= TEST_OUTPUT_QUIET=true ${EXTENDEDTEST} "--ginkgo.focus=${pf}" "--ginkgo.skip=${ps}" --ginkgo.dryRun --ginkgo.noColor | grep ok | grep -v skip | cut -c 20- | sort
-TEST_REPORT_DIR= TEST_OUTPUT_QUIET=true ${EXTENDEDTEST} "--ginkgo.focus=${sf}" "--ginkgo.skip=${ss}" --ginkgo.dryRun --ginkgo.noColor | grep ok | grep -v skip | cut -c 20- | sort
-echo
-
```

Comment:
this is only -dry-run that will show you which tests will run... however we have 0 skipped tests right now, which makes the sort fail because ginkgo fails when it fails to find the match... @stevekuznetsov and I agreed that we will remove this instead of going to fix the sorting logic here which basically just tells upfront what tests will run